### ISSUE TITLE
Mac

### ISSUE THREAD

**Issue by napereira** [briis/unifiprotect#6]
Video works on a raspberry pi, but doesn't appear to be working on a Mac hosting Home Assistant. Only getting thumbnails.

**Comment by briis** [comment]
Could you give me a description of your Home Assistant setup: VENV on Mac, Docker?

**Comment by napereira** [comment]
Sure. It's a VM python install on MacOS 10.13.  Accessing home assistant from any interface displays camera stills instead of video stream.

**Comment by briis** [comment]
Have you tried adding the `stream:` component to your `configuration.yaml`?

**Comment by briis** [comment]
Hi again,
Just installed the same environment as you, on a Mac. And found that without the `stream` component loaded I got the same result. But with `stream` loaded it works fine.
Had an issue with getting stream to work, as it tried to install PyAV, which failed to install on the Mac. What I did, from a terminal, is:
`brew install ffmpeg`
`brew link --overwrite ffmpeg`
`pip3 install av==6.1.2`
After that `stream` could be added to the configuration.yaml, and the streaming from the Camera works.
